### DISCUSSION TITLE
Kaldi model: reazonspeech-k2-v2

### DISCUSSION THREAD

**Discussion by dgoryeo** [SubtitleEdit/subtitleedit#8911] [Ideas]
Hi,

Is it possible to use this model as part of Kaldi/Vosk transcription framework: [reazonspeech-k2-v2](https://huggingface.co/reazon-research/reazonspeech-k2-v2) ;
`This model provides end-to-end Japanese speech recognition based on [Next-gen Kaldi](https://k2-fsa.org/).`

**Comment by niksedk** [comment]
Do they has Whisper models?
Perhaps you could try this? https://huggingface.co/japanese-asr

**Comment by dgoryeo** [comment]
Yes, I have used Kotoba which is a distilled Whisper. But I'd like to use a
Kaldi model as the title suggests.
